Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -240,20 +240,30 @@ public enum SymbolicLinkFlag : uint
         public Type OpType { get; private set; }
 
         /// <summary>
-        /// Path to file
+        /// Path to file. NOTE: This will not be accessable within TestProcess.exe
         /// </summary>
         public FileOrDirectoryArtifact Path { get; private set; }
 
+        /// <summary>
+        /// Should only be used in the context of TestProcess.exe
+        /// </summary>
+        private string PathAsString { get; set; }
+
         /// <summary>
         /// Content to write to file
         /// </summary>
         public string Content { get; private set; }
 
         /// <summary>
-        /// Path of new symlink or hardlink to create
+        /// Path of new symlink or hardlink to create. NOTE: This will not be accessable within TestProcess.exe
         /// </summary>
         public FileOrDirectoryArtifact LinkPath { get; private set; }
 
+        /// <summary>
+        /// Should only be used in the context of TestProcess.exe
+        /// </summary>
+        private string LinkPathAsString { get; set; }

```

Comment:
>set [](start = 47, length = 3)

'private set' ? I don't think this field should be changeable outside this class

Reply:
The property itself is private. You cannot specify access modifiers on getters/setters when that's the case